pt/endpoints/data/resource.py

```diff
@@ -0,0 +1,53 @@
+from datetime import date
+from flask import jsonify, request, make_response
+from flask_restful import Resource
+from utils.logging import logging
+from utils.oauth import auth, g
+from .model import Data
+from ..address.model import AMI, History
+
+
+class DatasResource(Resource):
+    # pylint: disable=R0201
```

Comment:
What is the purpose of ` DatasResource(Resource)`?
(What kinds of Data?)

Reply:
Change into PowerDatasResource.
Fix in 924cca0.